doc/api/fs.md

```diff
@@ -1702,8 +1702,11 @@ The listener callback gets two arguments `(eventType, filename)`.  `eventType` i
 `'rename'` or `'change'`, and `filename` is the name of the file which triggered
 the event.
 
-Please note the listener callback is attached to the `'change'` event
-fired by [`fs.FSWatcher`][], but they are not the same thing.
+Note that on most platforms, `'rename'` is also emitted when a file is deleted or added.
+In other words, it is emitted whenever a filename appears or disappears in the directory.
```

Comment:
Nit: add `on most platforms` here too to be as explicit as possible: `In other words, on most platforms, it is emitted...`